### ISSUE TITLE
Incorrect timestamp when unzipping universal apk

### ISSUE THREAD

**Issue by MartinGonzalez** [google/bundletool#316]
**Describe the bug**
When using command `bundletool.jar build-apks` the output zip, containing the apk, will provide an apk with the timestamp set in 1981.

**Bundletool version(s) affected**
Version: All

**Stacktrace**
Copy all of the output of the command, including the stacktrace if visible.

**To Reproduce**
* Run
```
java -jar ./bundletool.jar build-apks --bundle="./resources/example.aab" --output="./bundletool_temp/output.apks" --mode=universal 
```
* Extrack the universal apk 
* Check timestamp

**Expected behavior**
Expected to have at current date or at least the date the aab was created.

**Known workaround**
You can use `touch` command to update the Modified date.

<img width="237" alt="image" src="https://user-images.githubusercontent.com/4130008/219429529-e8a3a52b-0510-4df0-a2b7-94c9589675f5.png">

**Comment by slavdok** [comment]
I am experiencing the same issue.
Touch command updates "modified" date, but "created" date remains in 1981

**Comment by ymakhno** [comment]
`bundletool` does this intentionally to have deterministic output which allows to compare hashes of generated `.apks` file and ensure that output is the same.

Does it cause any problems for you?

**Comment by Martin-Gonzalez90** [comment]
> `bundletool` does this intentionally to have deterministic output which allows to compare hashes of generated `.apks` file and ensure that output is the same.
> 
> Does it cause any problems for you?

No, I'm not having problems with it, but just wanted to know if this was intentional or not because looked odd.
Thanks!

**Comment by Artman12** [comment]
I had a small issue where my company's storage automatically deleted old files, but I fixed it by changing timestamp
